docs/src/main/asciidoc/performance-measure.adoc

```diff
@@ -51,6 +51,32 @@ Likewise, don't fall into the trap of only measuring private memory which is wha
 While private memory might be useful in a environment deploying many different applications (and thus sharing memory a lot),
 it is very misleading in environments like Kubernetes/OpenShift.
 
+=== Measuring Memory Correctly on Docker
+
+In order to measure memory correctly **DO NOT use docker stat or anything derived from it** (e.g. ctop). This approach only measures a subset of the in-use resident pages, while the Linux Kernel, cgroups and cloud orchestration providers will utilize the full resident set in their accounting (determining whether a process is over the limits and should be killed). 
+
+To measure accurately, a similar set of steps for measuring RSS on Linux should be performed. The docker top command allows running a ps command on the container host machine against the processes in the container instance. By utilizing this in combination with formatting output parameters, the rss value can be returned: 
+
+[source,shell]
+----
+ $ docker ps # Copy the contianer id for the instance that is running
+ $ docker top <CONTAINER ID> -o pid,rss,args
+
+PID                 RSS                 COMMAND
+2531                27m                 ./application -Dquarkus.http.host=0.0.0.0
+----
+ 
+Alternatively, one can jump directly into a privleged shell (root on the host), and execute a ps command directly:
+
+[source]
+----
+ $ docker run -it --rm --privileged --pid=host justincormack/nsenter1
+ / # ps -e -o pid,rss,args | grep application
```

Comment:
Can't we have a one-liner with something like -c bash 'ps -e -o pid,rss,args | grep application`?

Reply:
This works:

```
docker run -it --rm --privileged --pid=host justincormack/nsenter1 /bin/bash -c 'ps -e -o pid,rss,args | grep application'
```

(be aware that the full path to bash must be set)

So, we can define an alias as follows:

```zsh
docker-rss() {
  docker run -it --rm --privileged --pid=host justincormack/nsenter1 /bin/bash -c ps -e -o pid,rss,args | grep $1 | awk '{$2=int($2/1024)"M";}{ print;}'
}
```

Reply:
`docker run` will start a new container, to run execute against a running container we would need to use `docker exec`

Reply:
but that's the trick, you start a privileged container accessing the host (justincormack/nsenter1).

Reply:
ah, I see I had assumed it was running the ps command on the process running in the container, nice

Reply:
Very cool trick

Reply:
@cescoffier I switched both options to single line statements , and shrunk the need for bash (/bin/ps)